Packages/com.unity.inputsystem/InputSystem/Plugins/Android/GameController.cs

```diff
@@ -0,0 +1,337 @@
+using System.Runtime.InteropServices;
+using ISX.Controls;
+using ISX.LowLevel;
+using ISX.Utilities;
+using UnityEngine;
+
+namespace ISX.Android
+{
+    [StructLayout(LayoutKind.Sequential)]

```

Comment:
`LayoutKind.Sequential` doesn't really do much. It's kinda what the code does anyway (at least on Mono). It's not enough to enforce layout to match between native and managed. From my experience so far, the only thing that really invariably guarantees matching layouts regardless of platform is `LayoutKind.Explicit`.

Reply:
For now, let's just leave it as is.